sandbox/src/mocks/updateAppointment/responses/MinimalBookingUpdateForCancel.json

```diff
@@ -0,0 +1,55 @@
+{
+  "id": "70000",
+  "meta": {
+    "versionId": "1",
+    "profile": [
+      "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-Appointment-1",
+      "https://fhir.hl7.org.uk/STU3/StructureDefinition/CareConnect-Appointment-1"
+    ]
+  },
+  "language": "en",
+  "resourceType": "Appointment",
+  "status": "cancelled",
+  "description": "e-Referral Appointment",
+  "start": "2021-07-11T12:00:00+00:00",
+  "end": "2021-07-11T13:00:00+00:00",
+  "reason": [
+    {
+      "coding": [
+        {
+          "system": "_baseurl_/STU3/CodeSystem/APPOINTMENT-CANCELLATION-REASON",
+          "code": "PATIENT_HOLIDAY",
+          "display": "Patient on holiday"
+        }
+      ]
+    }
+  ],
+  "comment": "Comments go here max 2000",
+  "created": "2021-06-30T13:02:32.133+00:00",
+  "incomingReferral": [
+    {
+      "reference": "ReferralRequest/000000070000/_history/7"
+    }
+  ],
+  "participant": [
+    {
+      "actor": {
```

Comment:
I wonder if we should think about this participant field for the service, have they still accepted the appointment given it has been cancelled?